src/modules/users/users.controller.ts

```diff
@@ -85,4 +89,19 @@ export class UsersController {
       data: updatedUser,
     };
   }
+
+  @Put('/wishlist/:accommodationId')
+  @UseGuards(JwtAuthGuard)
+  @UpdateUserSwaggerDecorator()
```

Comment:
why are you using UpdateUserSwaggerDecorator ? You are not updating user. You should create separate decorator with swagger documentation inside it.